### PR TITLE
Adding validation fields to JSON Output

### PR DESCRIPTION
**Pull request by kumarde**
This adds validation to JSON output, and a few unit tests that check if the resultant JSON can be parsed properly.

### CODE REVIEW

**Comment by kumarde** [comment]
@dadrian this one is also ready to be looked at!

**Comment by dadrian** [comment]
@kumarde I think I'm just going to take this one from here, thanks for the initial implementation.

**Comment by kumarde** [comment]
@dadrian okay no problem. Let me know if I can help in any other way.

**Comment by dadrian** [comment]
@zakird You want to give this a pass?

**Comment by zakird** [comment]
Yeah I'd like to. Can do first thing after lunch.

File: src/fastjson.cc [hunk 1]
```diff
@@ -205,40 +198,125 @@ void fast_dump_ct(std::ostream& f, const zsearch::CTStatus cts) {
...
+            } else {
+                f << ",";
+            }
+            fast_dump_path(f, p);
+        }
+        f << "]";
+    }
+    f << ",\"in_revocation_set\":"
+      << (rootStoreStatus.in_revocation_set() ? "true" : "false");
+    if (rootStoreStatus.parents_size() > 0) {
```
Comment (zakird) [inline src/fastjson.cc]: This should probably call `fast_dump_path`.

**Pull request merged**